### PR TITLE
Online schizophrenia: The 1 throwforcening

### PR DESCRIPTION
**Pull request by ReoDaProtovali**
## About The Pull Request
#7628 but I bent to the will of the devteam begrudgingly. the meat of the PR is still approved so that's cool. Who am I writing this for like 3 people outside of the same people who say the

The changelog is mostly just copied exept for the stuff I changed from the last one
## Changelog
:cl:
add: Added synth emotes to corrupt hounds and subtypes. *roboboop
add: Added a maw belly to leopardmanders that autotransfers to their gut. Dunno what made me do that (especially since it ended up being a lot of trouble to get working properly)
add: Added a whole new AI to swoopies based on nurse spiders looking for items to wrap. Right click an AI controlled swoopie and choose the "Change Settings" verb to tell the swoopie to swoop stuff!
add: Added wrappers dropping in vorebellies if you eat and digest the whole food item.
add: Added bones to merc and other humanoid corpses. (leave remains pref is now enabled for them!)
qol: Made vac-packs not ask if you want to set the destination to borg belly if you're not a borg
fix: Fixed throwing mobs into disposal bins. For some reason it was implemented but in a very shitty way that only checked for micro holders. It should work properly. No autoflush :(
image: added the secondary belly layer to the original set of vore bellies. Somehow we missed those and virgo had them. Now we should both have them!
/:cl:

### CODE REVIEW

**Comment by ReoDaProtovali** [comment]
i forgor a </span> 💀

**Comment by ReoDaProtovali** [comment]
suavemente

File: code/modules/recycling/disposal.dm [hunk 1]
```diff
@@ -528,14 +528,23 @@
 
 /obj/machinery/disposal/hitby(atom/movable/AM)
 	. = ..()
-	if(istype(AM, /obj/item) && !istype(AM, /obj/item/projectile))
+	//CHOMPEdit: disposal dunking and autoflushes with mobs~ - Reo
```
Comment (Kashargul) [inline code/modules/recycling/disposal.dm:531]: Only thing I see is this comment as leftover. Autoflushes are not in, so it shouldn't be mentioned that it does it.

Reword it to maybe: With the option to enable autoflush if desired

Comment (ReoDaProtovali) [reply code/modules/recycling/disposal.dm]: Changed the comment to reflect code changes. Honestly yeah I completely forgot this one.

**Comment by Raeschen** [comment]
Seems good. I might recommend a TM due to the amount of mechanical touches here just to check for instability

**Comment by Raeschen** [comment]
> Testmerge should be fine... If anything, im kinda iffy on the leopardmander addition in retrospect, since it might just end up being a bit more inconvinient, and I made it while brainrotting over them.

we got a backlog of TMs to get through first so you have time.
